tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for Pulp3 file system.
+
+:Requirement: Pulp3 installed
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: Pulp
```

Comment:
You'll need to add the metadata noting who owns this file

Reply:
OK, done